.maestro/release_tests/7_-_app_tp_onboarding.yaml

```diff
@@ -14,9 +14,6 @@ appId: com.duckduckgo.mobile.android
 - tapOn:
     text: "Settings"
     index: 0
-- scroll
-- scroll
-- scroll
```

Comment:
The scroll is not needed anymore as we moved this item in the upper part of the screen.